### PR TITLE
[COTCO-63][AC] Refactor template to iterate through

### PR DESCRIPTION
**Pull request by andreicostin**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -31,13 +31,13 @@ manager_view.total_staff=Total staff available
 manager_view.until_arrival=Estimated time until<br/> UK arrival (mins)
 manager_view.port_staff=Staff available
 manager_view.records=Transit records
-manager_view.until_1=60 +
-manager_view.until_2=45 - 60
-manager_view.until_3=30 - 44
-manager_view.until_4=15 - 29
-manager_view.until_5=Less than 15
+manager_view.until_0=60 +
+manager_view.until_1=45 - 60
```
Comment (robertbutacu) [inline conf/messages]: I think in the prototype, there was also the `minutes`

Comment (andreicostin) [reply conf/messages]: we removed the minutes from the cell

**Comment by peppe-hmrc** [comment]
Do we still need this? or it would be easier starting again?

**Comment by robertbutacu** [comment]
This can, indeed, be closed and deleted

**Pull request closed without merging**